stories/features/reboot.fmf

```diff
@@ -19,6 +19,11 @@ description: |
     option of the reboot script can be used, e.g. ``tmt-reboot -c
     "dnf system-upgrade reboot"``.
 
+    In some scenarios, rebooting may take longer than usual. The
+    default timeout used in tmt can be overriden using the ``-t``
+    option of the reboot script, e.g. ``tmt reboot -t 3600`` to
```

Comment:
This should be  ? I
``suggestion
    option of the reboot script, e.g. ``tmt-reboot -t 3600`` to
```

Reply:
Oh, good catch. Gosh, one more test suite run :/